Source/StyleCop.ruleset

```diff
@@ -1,101 +1,101 @@
+    <Rule Id="SA1001" Action="None" />
+    <Rule Id="SA1003" Action="None" />
+    <Rule Id="SA1005" Action="None" />
+    <Rule Id="SA1008" Action="None" />
+    <Rule Id="SA1009" Action="None" />
+    <Rule Id="SA1010" Action="None" />
+    <Rule Id="SA1011" Action="None" />
+    <Rule Id="SA1012" Action="None" />
+    <Rule Id="SA1013" Action="None" />
+    <Rule Id="SA1015" Action="None" />
+    <Rule Id="SA1025" Action="None" />
+    <Rule Id="SA1028" Action="None" />
+    <Rule Id="SA1100" Action="None" />
+    <Rule Id="SA1101" Action="None" />
+    <Rule Id="SA1107" Action="None" />
+    <Rule Id="SA1108" Action="None" />
+    <Rule Id="SA1111" Action="None" />
+    <Rule Id="SA1116" Action="None" />
+    <Rule Id="SA1117" Action="None" />
+    <Rule Id="SA1118" Action="None" />
+    <Rule Id="SA1119" Action="None" />
+    <Rule Id="SA1120" Action="None" />
+    <Rule Id="SA1121" Action="None" />
+    <Rule Id="SA1122" Action="None" />
+    <Rule Id="SA1123" Action="None" />
+    <Rule Id="SA1124" Action="None" />
+    <Rule Id="SA1127" Action="None" />
+    <Rule Id="SA1128" Action="None" />
+    <Rule Id="SA1129" Action="None" />
+    <Rule Id="SA1130" Action="None" />
+    <Rule Id="SA1131" Action="None" />
+    <Rule Id="SA1132" Action="None" />
+    <Rule Id="SA1133" Action="None" />
+    <Rule Id="SA1134" Action="None" />
+    <Rule Id="SA1135" Action="None" />
+    <Rule Id="SA1136" Action="None" />
+    <Rule Id="SA1200" Action="None" />
+    <Rule Id="SA1201" Action="None" />
+    <Rule Id="SA1202" Action="None" />
+    <Rule Id="SA1203" Action="None" />
+    <Rule Id="SA1204" Action="None" />
+    <Rule Id="SA1206" Action="None" />
+    <Rule Id="SA1208" Action="None" />
+    <Rule Id="SA1209" Action="None" />
+    <Rule Id="SA1210" Action="None" />
+    <Rule Id="SA1211" Action="None" />
+    <Rule Id="SA1212" Action="None" />
+    <Rule Id="SA1214" Action="None" />
+    <Rule Id="SA1300" Action="None" />
+    <Rule Id="SA1302" Action="None" />
+    <Rule Id="SA1303" Action="None" />
+    <Rule Id="SA1304" Action="None" />
+    <Rule Id="SA1306" Action="None" />
+    <Rule Id="SA1307" Action="None" />
+    <Rule Id="SA1308" Action="None" />
+    <Rule Id="SA1309" Action="None" />
+    <Rule Id="SA1310" Action="None" />
+    <Rule Id="SA1311" Action="None" />
+    <Rule Id="SA1312" Action="None" />
+    <Rule Id="SA1313" Action="None" />
+    <Rule Id="SA1314" Action="None" />
+    <Rule Id="SA1400" Action="None" />
+    <Rule Id="SA1401" Action="None" />
+    <Rule Id="SA1402" Action="None" />
+    <Rule Id="SA1403" Action="None" />
+    <Rule Id="SA1404" Action="None" />
+    <Rule Id="SA1405" Action="None" />
+    <Rule Id="SA1407" Action="None" />
+    <Rule Id="SA1408" Action="None" />
+    <Rule Id="SA1411" Action="None" />
+    <Rule Id="SA1413" Action="None" />
+    <Rule Id="SA1500" Action="None" />
+    <Rule Id="SA1501" Action="None" />
+    <Rule Id="SA1502" Action="None" />
+    <Rule Id="SA1503" Action="Error" />
```

Comment:
Just a quick comment. I think that by default whichever rule is omitted here, it gets checked.
So why would you leave this one in (and set `Action` to `Error`) instead of simply omitting it?

Reply:
Also, I found comments next to each rule to be kind of useful when trying to quickly figure out why something is failing. Why did you remove them? Thx!

Reply:
> Just a quick comment. I think that by default whichever rule is omitted here, it gets checked.
So why would you leave this one in (and set Action to Error) instead of simply omitting it?

The default is to emit a warning, not an error. But on second thought I think a warning is better. We can configure the PR build to treat warnings as errors, and then during development a warning won't block running the program.

> Also, I found comments next to each rule to be kind of useful when trying to quickly figure out why something is failing. Why did you remove them? Thx!

Oops, that was unintentional, thanks for spotting it!